frontend/src/metabase/visualizations/shared/utils/sizes.ts

```diff
@@ -0,0 +1,118 @@
+import _ from "underscore";
+import { DEFAULT_CARD_SIZE, GRID_WIDTH } from "metabase/lib/dashboard_grid";
+import { CardDisplayType } from "metabase-types/api";
+
+type VisualizationSize = { width: number; height: number };
+const VISUALIZATION_SIZES: {
```

Comment:
I moved it here thinking that, from my perspective at least, that it would be nicer to have all of the sizes in one place, so if we decided that we wanted to change a few of the sizes then we could do it here and not have to go through a bunch of files. idk if that makes much sense but that's just how I approached it

Otherwise I'll revert the change and put things back in their respective files

Reply:
I agree with @ranquild that if someone 6 months from now wants to change the size of the pie chart, they're probably going to look at the code for the pie chart visualization, so it makes more sense to have it there.

Having them all in one data structure also does make a certain amount of sense though. What if we imported this map into the individual Visualization components so we can have our cake and eat it too?

Reply:
Yeah I think that's a good solution, I'll make those changes